`panic: runtime error: index out of range` with v0.54.0

FYI: I get a panic error as shown below when I upgraded to `v0.54.0`, I downgraded to `v0.53.2` and everything worked fine again. I had been using `v0.53.0` previously.

```shell
HELMFILE_VERSION=0.54.0
echo "* Install helmfile:"
curl --silent --fail --retry 3 --location https://github.com/roboll/helmfile/releases/download/v"${HELMFILE_VERSION}"/helmfile_linux_amd64 --output /usr/local/bin/helmfile
chmod +x /usr/local/bin/helmfile

helmfile --version
  helmfile version v0.54.0
```

```shell
helmfile --log-level=debug --file=./ --selector=enabled=true,infra=true,app=cert-manager lint
```

```
panic: runtime error: index out of range [recovered]
	panic: runtime error: index out of range

goroutine 1 [running]:
github.com/roboll/helmfile/vendor/github.com/urfave/cli.HandleAction.func1(0xc000136e30)
	/go/src/github.com/roboll/helmfile/vendor/github.com/urfave/cli/app.go:474 +0x278
panic(0x813dc0, 0xbc1b70)
	/usr/local/go/src/runtime/panic.go:513 +0x1b9
github.com/roboll/helmfile/state.(*HelmState).loadEnv(0xc00008a600, 0x87e044, 0x7, 0x8994e0, 0xc00004e040, 0x1, 0x1)
	/go/src/github.com/roboll/helmfile/state/create.go:161 +0x131b
github.com/roboll/helmfile/state.(*creator).CreateFromYaml(0xc000136198, 0xc000176000, 0x55b, 0x69f, 0xc000084920, 0x11, 0x87e044, 0x7, 0xc000000300, 0x200000003, ...)
	/go/src/github.com/roboll/helmfile/state/create.go:110 +0x897
github.com/roboll/helmfile/pkg/app.(*twoPassRenderer).renderEnvironment(0xc000136580, 0xc000164000, 0x58a, 0x78a, 0xc0001362c0, 0x49c555, 0xc0000887e0)
	/go/src/github.com/roboll/helmfile/pkg/app/two_pass_renderer.go:49 +0x2e2
github.com/roboll/helmfile/pkg/app.(*twoPassRenderer).renderTemplate(0xc000136580, 0xc000164000, 0x58a, 0x78a, 0x78a, 0x0, 0x0)
	/go/src/github.com/roboll/helmfile/pkg/app/two_pass_renderer.go:65 +0x6a
github.com/roboll/helmfile/pkg/app.(*App).VisitDesiredStates.func1(0xc000084920, 0x11, 0xc000136628, 0x4441f7)
	/go/src/github.com/roboll/helmfile/pkg/app/app.go:128 +0x166
github.com/roboll/helmfile/pkg/app.(*App).visitStateFiles.func1(0x2, 0x2)
	/go/src/github.com/roboll/helmfile/pkg/app/app.go:101 +0x37
github.com/roboll/helmfile/pkg/app.(*App).within(0xc000158090, 0x87cc8c, 0x1, 0xc0001367f8, 0x2, 0x2)
	/go/src/github.com/roboll/helmfile/pkg/app/app.go:48 +0x60b
github.com/roboll/helmfile/pkg/app.(*App).visitStateFiles(0xc000158090, 0x7ffeea725e2a, 0x2, 0xc000136880, 0xc00008d8f0, 0x0)
	/go/src/github.com/roboll/helmfile/pkg/app/app.go:100 +0x272
github.com/roboll/helmfile/pkg/app.(*App).VisitDesiredStates(0xc000158090, 0x7ffeea725e2a, 0x2, 0xc0001368e0, 0x861360, 0xc000136908)
	/go/src/github.com/roboll/helmfile/pkg/app/app.go:114 +0x87
github.com/roboll/helmfile/pkg/app.(*App).VisitDesiredStatesWithReleasesFiltered(0xc000158090, 0x7ffeea725e2a, 0x2, 0xc000093220, 0x2, 0x0)
	/go/src/github.com/roboll/helmfile/pkg/app/app.go:197 +0x97
github.com/roboll/helmfile/cmd.FindAndIterateOverDesiredStatesUsingFlagsWithReverse(0xc000102c80, 0x0, 0xc00008f3f0, 0xbd1b01, 0xc00008f3f0)
	/go/src/github.com/roboll/helmfile/cmd/cmd.go:79 +0xe1
main.findAndIterateOverDesiredStatesUsingFlags(0xc000102c80, 0xc00008f3f0, 0x16, 0xc0000931c0)
	/go/src/github.com/roboll/helmfile/main.go:598 +0x3a
main.main.func7(0xc000102c80, 0x0, 0x0)
	/go/src/github.com/roboll/helmfile/main.go:335 +0x60
reflect.Value.call(0x7f8a20, 0x899548, 0x13, 0x87d178, 0x4, 0xc000136dd0, 0x1, 0x1, 0xc000098000, 0xc0000a7770, ...)
	/usr/local/go/src/reflect/value.go:447 +0x449
reflect.Value.Call(0x7f8a20, 0x899548, 0x13, 0xc000136dd0, 0x1, 0x1, 0x5, 0x4, 0xc0000b5500)
	/usr/local/go/src/reflect/value.go:308 +0xa4
github.com/roboll/helmfile/vendor/github.com/urfave/cli.HandleAction(0x7f8a20, 0x899548, 0xc000102c80, 0x0, 0x0)
	/go/src/github.com/roboll/helmfile/vendor/github.com/urfave/cli/app.go:483 +0x1fb
github.com/roboll/helmfile/vendor/github.com/urfave/cli.Command.Run(0x87d638, 0x5, 0x0, 0x0, 0x0, 0x0, 0x0, 0x89279e, 0x3f, 0x0, ...)
	/go/src/github.com/roboll/helmfile/vendor/github.com/urfave/cli/command.go:186 +0x8f6
github.com/roboll/helmfile/vendor/github.com/urfave/cli.(*App).Run(0xc00008a300, 0xc00008c000, 0x3, 0x3, 0x0, 0x0)
	/go/src/github.com/roboll/helmfile/vendor/github.com/urfave/cli/app.go:237 +0x52e
main.main()
	/go/src/github.com/roboll/helmfile/main.go:557 +0x200e
```

--- a/state/create.go
+++ b/state/create.go
@@ -158,7 +158,15 @@ func (st *HelmState) loadEnv(name string, readFile func(string) ([]byte, error))
 				if _, err := os.Stat(path); os.IsNotExist(err) {
 					return nil, err
 				}
-				release := &st.Releases[0]
+				// Work-around to allow decrypting environment secrets
+				//
+				// We don't have releases loaded yet and therefore unable to decide whether
+				// helmfile should use helm-tiller to call helm-secrets or not.
+				//
+				// This means that, when you use environment secrets + tillerless setup, you still need a tiller
+				// installed on the cluster, just for decrypting secrets!
+				// Related: https://github.com/futuresimple/helm-secrets/issues/83
+				release := &ReleaseSpec{}
 				flags := st.appendTillerFlags([]string{}, release)
 				decFile, err := helm.DecryptSecret(st.createHelmContext(release, 0), path, flags...)
 				if err != nil {

